docs/developer-guide/checks.md

```diff
@@ -5,9 +5,15 @@ Here you can find how to create new checks for Prowler.
 **To create a check is required to have a Prowler provider service already created, so if the service is not present or the attribute you want to audit is not retrieved by the service, please refer to the [Service](./services.md) documentation.**
 
 ## Introduction
+
+The checks are the foundamental piece of Prowler. A check is a simply piece of code that ensure if something is configured whith best cybersecurity practices, or is not and could be a breach of a cloud infrastructure. Furthemore, some information about the audition that is being doing over that resource (*metadata*).
```

Comment:
```suggestion
The checks are the foundamental piece of Prowler. A check is a simply piece of code that ensures if something is configured against cybersecurity best practices.
```

What do you want to say with `Furthemore, some information about the audition that is being doing over that resource (*metadata*).`?

Reply:
I was wanting to say that Checks are the piece of code and metadata about the piece of code. But reading now it could be a little bit confusing, I am going to rewrite it.